src/errors.js

```diff
@@ -1,8 +1,10 @@
-var NO_DATA_MSG = 'No data'
```

Comment:
Is it gone? Why?

Reply:
https://github.com/santiment/google-spreadsheet-addon/pull/82/files/201421bf2bb4539e9bba47a4ce267052107ba47d#diff-5c8b199a98cb05fcfa67ad0a331c165bR3

it is just a const now

Reply:
Nevermind